docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -13,42 +13,55 @@ The sample configuration files below run Semgrep CI on various continuous integr
 ## GitHub Actions
 
 ```yaml
+# Name of this GitHub Actions workflow.
 name: Semgrep
 
 on:
-  # Scan changed files in PRs, block on new issues only (existing issues ignored)
+  # Determine when you want Semgrep to scan your code.
+  # Use as many of the following options as you want.
+  #
+  # Option 1: Scan changed files in PRs, only report new findings (existing
+  # findings in the repository are ignored).
+  # To run on specific types of PR states (opened, reopened, etc) or particular
+  # paths or branches, see the following GitHub documentation:
+  # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request
   pull_request: {}
 
-  # Scan all files on branches, block on any issues
+  # Option 2: Scan all files on branches, report any findings.
   # push:
   #   branches: ["master", "main"]
 
-  # Schedule this job to run at a certain time, using cron syntax
-  # Note that * is a special character in YAML so you have to quote this string
+  # Option 3: Schedule CI job to run at a certain time, using cron syntax.
+  # Note: the asterisk sign * is a special character in YAML so you have to quote this string
   # schedule:
   #   - cron: '30 0 1,15 * *' # scheduled for 00:30 UTC on both the 1st and 15th of the month
```

Comment:
Maybe we uncomment this? That way if people copied, it's still our recommended config
```suggestion
  schedule:
    - cron: '30 0 1,15 * *' # scheduled for 00:30 UTC on both the 1st and 15th of the month
```

Reply:
yes!